source/slang/slang-ir-inst-defs.h

```diff
@@ -566,6 +571,7 @@ INST(SwizzledStore, swizzledStore, 2, 0)
 /* IRTerminatorInst */
 
     INST(Return, return_val, 1, 0)
+    INST(Yield, yield, 1, 0)
```

Comment:
I just remembered that c# uses the keyword, "yield" for something else. It is similar to how async keyword works for java-script.
From my experience from game engines, the word, "yield", is more often used for thread related control.
I am not sure if it will be good to go with the keyword "yield" for the variadic template feature.

Reply:
As discussed offline, I believe the yield here has very similar meaning to c#'s yield, so keeping this name.